code/src/beetle.cpp

```diff
@@ -28,47 +31,86 @@ namespace r2d2::moving_platform {
         // is made for -10 till 10.
         else if (speed < 10 && speed > -10) {
             speed = 0;
-            qik_2s12v10_motorcontroller.set_m0_speed(0);
-            qik_2s12v10_motorcontroller.set_m1_speed(0);
         }
+        qik_2s12v10_motorcontroller.set_m0_speed(speed);
+        qik_2s12v10_motorcontroller.set_m1_speed(-speed);
     }
 
     void beetle_c::set_steering(int16_t degrees) {
     }
     void beetle_c::turn(int16_t degrees) {
-        int new_degrees = degrees;
         // because of inaccuracies, when moving forward, the steer of manual
         // control will give a number beween -10 and 10 Just to make sure the
         // robot will move forward and not react to quickly, a theshold is made
         // for -15 till 15.
-        if (new_degrees < 15 && new_degrees > -15) {
-            new_degrees = 0;
+        if (degrees >= 15 && degrees <= 90) {
+            degrees = degrees;
```

Comment:
does anything even change here?

Reply:
variable name, we don't use new_degrees anymore